initializr-generator/src/test/java/io/spring/initializr/util/VersionRangeTests.java

```diff
@@ -123,6 +123,30 @@ public void matchOverAsOfCurrentVersion() {
 						Version.parse("1.3.6.BUILD-SNAPSHOT")))));
 	}
 
+	@Test
+	public void toVersionRangeWithSimpleVersion() {
+		VersionRange range = new VersionParser(
+				Collections.singletonList(Version.parse("1.5.6.RELEASE")))
+						.parseRange("1.3.5.RELEASE");
+		assertThat(range.toVersionRange()).isEqualTo("1.3.5.RELEASE");
+	}
+
+	@Test
+	public void toVersionRangeWithVersionsIncluded() {
+		VersionRange range = new VersionParser(
+				Collections.singletonList(Version.parse("1.5.6.RELEASE")))
+						.parseRange("[1.3.5.RELEASE,1.5.5.RELEASE]");
+		assertThat(range.toVersionRange()).isEqualTo("[1.3.5.RELEASE,1.5.5.RELEASE]");
+	}
+
+	@Test
+	public void toVersionRangeWithVersionExcluded() {
```

Comment:
Is it worth having another test for an exclusive upper-bound on the range?

Reply:
yeah, don't know why I left that one out 😕